cmd/astro-wasm/astro-wasm.go

```diff
@@ -1,4 +1,4 @@
-//go:build js && wasm
```

Comment:
what does this change do?

Reply:
I think this is go 1.17 syntax? Not sure totally, just that the wasm won't build with this old comment.